src/styles/variables.css

```diff
@@ -20,6 +20,7 @@
   --grey-blue-1: rgb(232, 236, 245);
   --grey-blue-2: rgb(244, 246, 250);
   --grey-blue-3: rgb(246, 246, 246);
+  --light-blue: rgb(238,242,245);
```

Comment:
Two things here:
- Please add this next to the other blue colors _(eg: `--colony-blue`)_
- Please add spaces between the rgb values so we have consistent color declrations